src/ansys/tools/visualization_interface/backends/pyvista/widgets/screenshot.py

```diff
@@ -69,8 +71,12 @@ def callback(self, state: bool) -> None:
 
     def update(self) -> None:
         """Define the configuration and representation of the screenshot widget button."""
+        if self._dark_mode:
+            is_inv = "_inv"
+        else:
+            is_inv = ""
         show_vr = self._button.GetRepresentation()
-        show_icon_file = Path(Path(__file__).parent / "_images" / "screenshot.png")
+        show_icon_file = Path(Path(__file__).parent / "_images" / ("screenshot" + is_inv + ".png"))
```

Comment:
I'd prefer the usage of f-strings rather than string concatenation. It's more modern Python that way, but feel free to disregard

Reply:
Agree with all your comments, fixed in dff7771397ee0b813f15f207ecff48843b8c98bb